scripts/cpplint.py

```diff
@@ -550,9 +550,13 @@
   "Remove space before ( in switch (": "s/switch (/switch(/",
   "Should have a space between // and comment": 's/\/\//\/\/ /',
   "Missing space before {": r's/\([^ ]\){/\1 {/',
-  "Tab found, replace by spaces": r's/\t/  /',
+  "Tab found, replace by spaces": r's/\t/  /g',
   "Line ends in whitespace.  Consider deleting these extra spaces.": r's/\s*$//',
-  #"Redundant blank line at the end of a code block should be deleted.": "d", # messes up line numbers for other errors.
+  "You don't need a ; after a }": r's/};/}/',
+  "Missing space after ,": r's/,\([^ ]\)/, \1/',
```

Comment:
It should be safe to apply this one globally, i.e., add a 'g' at the end of the pattern.